templates/mon.sql.j2

```diff
@@ -7,7 +7,10 @@ CREATE TABLE `alarm` (
   `id` varchar(36) COLLATE utf8mb4_unicode_ci NOT NULL,
   `alarm_definition_id` varchar(36) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
   `state` enum('UNDETERMINED','OK','ALARM') COLLATE utf8mb4_unicode_ci NOT NULL,
+  `lifecycle_state` varchar(50) DEFAULT NULL,
```

Comment:
Maybe it would make sense to allow it to be null until everything gets in and then later change it to NOT NULL. Otherwise this and the threshold engine change have to go in at the same time or mini-mon will be broken. And we don't how long it will take zuul to publish the new jar